bears/vcs/git/GitCommitBear.py

```diff
@@ -109,6 +110,35 @@ def check_shortlog(self,
                 yield Result(
                     self,
                     "Shortlog of HEAD commit does not match given regex.")
+        colon_pos = shortlog.find(':')
```

Comment:
would be good if you introduce a `\n` before this one, as this is new functionality, makes it easier to read^^